chatcommands.py

```diff
@@ -1735,13 +1736,28 @@ def allspam(msg, url):
         batch = ""
         if len(user_posts) > 1:
             batch = " (batch report: post {} out of {})".format(index, len(user_posts))
+        if has_already_been_posted(post.site, post.post_id, post.title) and not is_false_positive(
+                (post.post_id, post.site)):
+            if GlobalVars.metasmoke_key is not None:
```

Comment:
This should be:

    if GlobalVars.metasmoke_key and GlobalVars.metasmoke_host:

alues appear to be